fabfile.py

```diff
@@ -161,6 +153,30 @@ def test(c, lint=False, parallel=False):
 # -----------------------------------------------------------------------------
 
 
+@task
+def create_superuser(c):
+    """
+    Run bash in a local container (with access to dependencies)
+    """
+    subprocess.run(
+        [
+            "docker-compose",
+            "exec",
+            "web",
+            "poetry",
+            "run",
+            "python",
+            "manage.py",
+            "createsuperuser",
+        ]
+    )
```

Comment:
I've tried running `fab create_superuser` but it doesn't seem to work - do I need to run a different command to get this working?

Reply:
Brilliant, yep, that's the one!